nixos/modules/system/boot/tmp.nix

```diff
@@ -34,6 +34,7 @@ with lib;
       {
         what = "tmpfs";
         where = "/tmp";
+        type = "tmpfs";
         mountConfig.Options = [ "mode=1777" "strictatime" "rw" "nosuid" "nodev" "size=50%" ];
```

Comment:
This is where we differ from upstream: we don't have an inode limit, whereas upstream does:

https://github.com/systemd/systemd/blob/4d484e14bb9864cef1d124885e625f33bf31e91c/units/tmp.mount#L25

Reply:
Yeah I saw that after I posted the PR and forgot to update 😅, although that has been there since May, so probably isn't something that is worth us forking it for IMO (As most people will never run into it)